### PR TITLE
Add support for gmetad/gmond params

### PR DESCRIPTION
**Pull request by eabecho**
Fix CRLF issue.
Fix gmetad_package_name
Fix params for RHEL 5 (for current EPEL packages).

### CODE REVIEW

**Comment by eabecho** [comment]
Working on fixing tests.

**Comment by eabecho** [comment]
Spec'd rake version in Gemfile.

File: manifests/gmetad.pp [hunk 1]
```diff
@@ -13,7 +13,8 @@
   $gmetad_service_config           = $::ganglia::params::gmetad_service_config,
   $gmetad_user                     = $::ganglia::params::gmetad_user,
   $gmetad_case_sensitive_hostnames =
-    $::ganglia::params::gmetad_case_sensitive_hostnames
+    $::ganglia::params::gmetad_case_sensitive_hostnames,
```
Comment (jhoblitt) [inline manifests/gmetad.pp]: indentation

File: manifests/gmond.pp [hunk 2]
```diff
@@ -36,6 +39,8 @@
   if !(is_string($gmond_package_name) or is_array($gmond_package_name)) {
     fail('$gmond_package_name is not a string or array.')
   }
+  validate_string($gmond_service_name)
+  validate_string($gmond_service_config)
```
Comment (jhoblitt) [inline manifests/gmond.pp]: `$gmond_status_command` should be validated.

File: CHANGELOG.md [hunk 3]
```diff
@@ -352,8 +352,8 @@ gmond.conf
 
  * [972e5ad](../../commit/972e5ad) - __(Jonathan Beckman)__ Removed lies from gmetad config for el6
 
-By default EL6 ganglia runs as nobody and not ganglia.
-This fixes the issue by changing the user that ganglia is
+By default EL6 ganglia runs as nobody and not ganglia.
+This fixes the issue by changing the user that ganglia is
```
Comment (jhoblitt) [inline CHANGELOG.md]: Why is the CHANGELOG being modified?

File: manifests/params.pp [hunk 4]
```diff
@@ -65,11 +65,11 @@
           case $::operatingsystemmajrelease {
             # the epel packages change uid/gids + install paths between 5 & 6
             '5': {
-              $gmond_service_config = '/etc/gmond.conf'
+              $gmond_service_config = '/etc/ganglia/gmond.conf'
```
Comment (jhoblitt) [inline manifests/params.pp]: Why is this being changed?

File: CHANGELOG.md (hunk 3 above)
Comment (eabecho) [inline CHANGELOG.md]: There's a CRLF issue that git kept flagging.  I just ran that file through dos2unix.

File: manifests/params.pp (hunk 4 above)
Comment (eabecho) [inline manifests/params.pp]: The current EPEL ganglia binaries now use /etc/ganglia/gmond.conf.

File: manifests/gmetad.pp (hunk 1 above)
Comment (eabecho) [inline manifests/gmetad.pp]: I just added a comma on this line.  What would you like me to do?

**Comment by eabecho** [comment]
Ping.

File: manifests/gmetad.pp (hunk 1 above)
Comment (jhoblitt) [inline manifests/gmetad.pp]: Aw, it's fine as-is.

**Comment by jhoblitt** [comment]
@eabecho Pong ;)

**Comment by eabecho** [comment]
Thanks.  Is there a release pending?

**Pull request merged**